src/api/python/cvc5.pxi

```diff
@@ -1478,16 +1504,39 @@ cdef class Solver:
             raise ValueError("Unexpected inputs to mkBitVector")
         return term
 
-    def mkFiniteFieldElem(self, value, Sort sort):
+    def mkFiniteFieldElem(self, value, Sort sort, *args):
         """
             Create finite field value.
 
+            Supports the following arguments:
+            
+            - ``Term mkFiniteFieldElem(value, Sort sort)``
+            - ``Term mkFiniteFieldElem(string value, Sort sort, int base)``
+
             :return: A Term representing a finite field value.
-            :param value: The value of the element's integer representation.
+            :param value: The value of the element's integer or string in base 10 representation.
```

Comment:
```suggestion
            :param value: The value of the element's integer or string (in base
                                      10 representation if no ``base`` given).
```
It seems that this is the behavior you described above? Maybe also add a test for the case when it is not base 10? For example, a hex value?

Reply:
I can simply delete functions like `Sort Solver::mkFiniteFieldSort(const std::string& modulus) const` and keep the `Sort Solver::mkFiniteFieldSort(const std::string& modulus, uint32_t base = 10) const`. Don't know why I didn't suggest it before.

Reply:
And also add the tests for the default base

Reply:
Sounds good!